test/use-swr-error.test.tsx

```diff
@@ -1,19 +1,23 @@
 import { act, fireEvent, render, screen } from '@testing-library/react'
 import React, { useEffect, useState } from 'react'
-import useSWR, { mutate } from 'swr'
-import { sleep, createResponse } from './utils'
+import useSWR from 'swr'
+import { sleep, createResponse, TestSWRConfig } from './utils'
 
 describe('useSWR - error', () => {
   it('should handle errors', async () => {
     function Page() {
-      const { data, error } = useSWR('error-1', () =>
+      const { data, error } = useSWR('error-key', () =>
         createResponse(new Error('error!'))
       )
       if (error) return <div>{error.message}</div>
       return <div>hello, {data}</div>
     }
 
-    render(<Page />)
+    render(
+      <TestSWRConfig>
```

Comment:
I dont think we should replace this with wrapping context provider. Most of the tests are also to make sure they're good to run without context provider. We shouldn't replace unique key with a standalone cache in context.

Reply:
Thank you for your feedback! That's a good point.
Is the point solved by adding tests with a global cache or should we confirm to pass existing test cases with a global cache?

Reply:
> Is the point solved by adding tests with a global cache or should we confirm to pass existing test cases with a global cache?

Yeah that's better 👍